src/components/v2/shared/DistributionSplitsSection/DistributionSplitModal.tsx

```diff
@@ -187,7 +201,18 @@ export default function DistributionSplitModal({
 
   // Validates new payout receiving address
   const validatePayoutAddress = () => {
-    return validateEthAddress(beneficiary ?? '', splits, mode, splitIndex)
+    if (
+      editableSplitIndex !== undefined &&
+      beneficiary === editableSplits[editableSplitIndex].beneficiary
+    ) {
```

Comment:
We have a validation to prevent adding duplicate addresses for payouts. This is so we can still submit edited payouts without triggering this error